chunkSize logic is off by 1

if chunkSize <= minChunkSize in s3.go 

should be:
if chunkSize < minChunkSize

otherwise this:
panic: The chunksize 5242880 parameter should be a number that is larger than or equal to 5242880

--- a/registry/storage/driver/s3/s3.go
+++ b/registry/storage/driver/s3/s3.go
@@ -168,7 +168,7 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 			return nil, fmt.Errorf("invalid valud for chunksize: %#v", chunkSizeParam)
 		}
 
-		if chunkSize <= minChunkSize {
+		if chunkSize < minChunkSize {
 			return nil, fmt.Errorf("The chunksize %#v parameter should be a number that is larger than or equal to %d", chunkSize, minChunkSize)
 		}
 	}

